Tools/scripts/deepfreeze.py

```diff
@@ -358,6 +365,7 @@ def generate(self, name: str, obj: object) -> str:
             val = self.generate_tuple(name, obj)
         elif isinstance(obj, str):
             val = self.generate_unicode(name, obj)
+            self.patchups.append(f"INTERN_STRING({val});")
```

Comment:
You should only call this for strings that look like identifiers (see all_name_chars() in codeobject.c).